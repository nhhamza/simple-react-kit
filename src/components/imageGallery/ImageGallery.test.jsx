import React from 'react';

import { shallow } from 'enzyme';

import Gallery from './ImageGallery.jsx';

describe('Image Gallery component', () => {
  const wrapper = shallow(<Gallery />);
  it('should render', () => {
    expect(wrapper.exists()).toMatchSnapshot();
  });
});
