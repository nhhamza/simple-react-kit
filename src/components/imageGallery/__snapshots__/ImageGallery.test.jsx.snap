// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Gallery component should render 1`] = `true`;
